core/src/main/scala/com/quantifind/charts/highcharts/Highchart.scala

```diff
@@ -288,7 +289,7 @@ case class Highchart(
         hckTraversableToServiceFormat(series) ++
         Seq(xAxis, yAxis).flatMap(optionArrayAxisToServiceFormat) ++
         optionArrayColorToServiceFormat(colorWrapper) ++
-        Seq(chart, title, exporting, credits, legend, tooltip, subtitle).flatMap(optionToServiceFormat)
+        Seq(chart, colorAxis, title, exporting, credits, legend, tooltip, subtitle).flatMap(optionToServiceFormat)
```

Comment:
Wow bad name for colorAxis, should've used colourAxis to be friendly to international speakers